src/torchmetrics/functional/classification/accuracy.py

```diff
@@ -378,12 +378,12 @@ def accuracy(
     threshold: float = 0.5,
     num_classes: Optional[int] = None,
     num_labels: Optional[int] = None,
-    average: Optional[Literal["micro", "macro", "weighted", "none"]] = "micro",
-    multidim_average: Optional[Literal["global", "samplewise"]] = "global",
+    average: Literal["micro", "macro", "weighted", "none"] = "micro",
+    multidim_average: Literal["global", "samplewise"] = "global",
     top_k: Optional[int] = 1,
     ignore_index: Optional[int] = None,
     validate_args: bool = True,
-) -> Tensor:
+) -> Optional[Tensor]:
```

Comment:
I think it shall always return only tensor r raise exception...
the same you may say about class with return None, this maybe NOne is confusing for user, and also any typing

Reply:
are you okay with me adding `type: ignore[return-value]` to ignore the error at that line?

Reply:
I think it is better option from the two

Reply:
implemented in https://github.com/Lightning-AI/torchmetrics/pull/1448/commits/82800cdebc71d5c976f8652ce7529657e5149910